### ISSUE TITLE
animating different element on hover

### ISSUE THREAD

**Issue by idkjs** [reasonml-labs/bs-css#190]
How would I implement this `css`:

```css
li:hover ~ li p {
  animation: wave-animation 0.3s infinite;
}
```

Right now I have this:

![Screen Recording 2020-03-28 at 7 54 53 PM](https://user-images.githubusercontent.com/2370391/77836640-5105f280-712e-11ea-9768-73edb2680e9c.gif)

What Im trying to do is make the hand shake when i hover the words.

Thanks.

**Comment by giraud** [comment]
something like 
```
  style([
     selector("li p", [
         hover([ animation(~duration=200, ~iterationCount=infinite, waveAnimation, ), ]),
      ],
 ), ])
```

with animationWave defined with keyframes

```
let waveAnimation =
  Css.(
    keyframes([
      (0, [transform(...)]),
      (100, [transform(...)]),
    ])
  );
```

?

**Comment by idkjs** [comment]
Thanks. I will try to implement. I have tried adding it here https://github.com/idkjs/bs-css-emotion-footer/blob/2a63e5fec8ca0c7b34782019e9313e1bc70c5967/src/Footer.re#L52 but that doesnt do the trick. It animates the hand when you hover on the hand. Will keep at it.

**Comment by giraud** [comment]
Yes, of course, only the hand is a 'p'.
if you put a style on each li, you can just use:
```
  let li =
    style([
      selector("li:lastChild", [gridColumn(1, 2), gridRow(1, 2)]),
      hover([animation(~duration=300, ~iterationCount=infinite, waveAnimation)]),
    ]);
```

or if you put a style on 'ul', an easy way to do it is:
```
  let ul =
    merge([
      style([
        display(`inlineGrid),
        gridAutoFlow(`row),
        gridGap(px(24)),
        justifyContent(`center),
        margin(`auto),
        children([hover([animation(~duration=300, ~iterationCount=infinite, waveAnimation)])]),
      ]),
      style([media("(min-width: 500px)", [gridAutoFlow(`column)])]),
    ]);

```

**Comment by idkjs** [comment]
Thanks for this feedback. I think I explained it wrong. The idea is that when you hover over the three items, they do not move but it triggers the `p` tag/hand to wave. So you want a hover to act like a function, i guess that triggers the wave css on the hand.

I got it to produce that result by moving the `selector` call inside of the `hover` call like so:

```ocaml
  let li =
    style([
      selector("li:lastChild", [gridColumn(1, 2), gridRow(1, 2)]),
      hover([
        selector(
          "~ li p",
          [
            animation(~duration=300, ~iterationCount=infinite, waveAnimation),
          ],
        ),
      ]),
    ]);
```

![Screen Recording 2020-03-31 at 10 28 08 AM](https://user-images.githubusercontent.com/2370391/78038117-723b2e80-733a-11ea-80c3-febf7e2f9096.gif)


The last thing I cant figure out is how to move the hand to the front of the line.

This is just a port of this [codepen](https://codepen.io/julesforrest/pen/qLpgNB).

It has the `hand` in the first position. It may be related to `justify-items` in the original css.

Is `justify-items: center;` the same as ```justifyContent(`center)```. I don't see `justify-items` in the api anywhere.

Thanks.
